logic/src/commonJvmAndroid/kotlin/com/wire/kalium/logic/feature/call/GlobalCallManager.kt

```diff
@@ -30,26 +30,34 @@ import com.wire.kalium.logic.data.call.CallRepository
 import com.wire.kalium.logic.data.call.VideoStateChecker
 import com.wire.kalium.logic.data.call.mapper.CallMapper
 import com.wire.kalium.logic.data.conversation.ConversationRepository
+import com.wire.kalium.logic.data.id.CurrentClientIdProvider
 import com.wire.kalium.logic.data.id.FederatedIdMapper
 import com.wire.kalium.logic.data.id.QualifiedID
 import com.wire.kalium.logic.data.id.QualifiedIdMapper
 import com.wire.kalium.logic.data.user.UserId
 import com.wire.kalium.logic.data.user.UserRepository
-import com.wire.kalium.logic.data.id.CurrentClientIdProvider
 import com.wire.kalium.logic.feature.call.usecase.ConversationClientsInCallUpdater
 import com.wire.kalium.logic.feature.message.MessageSender
 import com.wire.kalium.logic.featureFlags.KaliumConfigs
 import com.wire.kalium.logic.util.CurrentPlatform
 import com.wire.kalium.logic.util.PlatformContext
 import com.wire.kalium.logic.util.PlatformType
 import com.wire.kalium.network.NetworkStateObserver
+import com.wire.kalium.util.KaliumDispatcher
+import com.wire.kalium.util.KaliumDispatcherImpl
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.SupervisorJob
 import java.util.concurrent.ConcurrentHashMap
 import java.util.concurrent.ConcurrentMap
 
 actual class GlobalCallManager(
-    appContext: PlatformContext
+    appContext: PlatformContext,
+    kaliumDispatchers: KaliumDispatcher = KaliumDispatcherImpl
 ) {
 
+    private val job = SupervisorJob()
+    private val scope = CoroutineScope(job + kaliumDispatchers.io)
```

Comment:
It's better to use `globalKaliumScope` instead.

Just pass `globalScope: CoroutineScope` as a parameter.

`GlobalKaliumScope` is a CoroutineScope.

Currently we don't support "Stopping" the global scope. But if we ever do, it will automatically stop the Calling mechanisms as well.